### DISCUSSION TITLE
AWS terraformer import filters does not work

### DISCUSSION THREAD

**Discussion by jobinpresidio** [GoogleCloudPlatform/terraformer#1120] [Q&A] [answered]
terraformer import aws --resources=rds --filter="Type=db_instance;Name=tags.name;Value=testing" --regions=us-east-1 
18:08:42 aws importing region us-east-1
 18:08:47 aws importing... rds
18:08:51 aws done importing rds
18:08:51 Number of resources for service rds: 35

Trying to list a single resource, but all the rds resources are imported 🧐
can anyone help me out with this

**Comment by sunil29feb** [comment, marked as answer]
Give a try removing Type and its value - 
terraformer import aws --resources=rds --filter="Name=tags.name;Value=testing" --regions=us-east-1
